projects/ngrx.io/content/examples/router-store-selectors/src/app/router.selectors.ts

```diff
@@ -0,0 +1,17 @@
+// #docregion routerSelectors
+import { getSelectors, RouterReducerState } from '@ngrx/router-store';
+import { createFeatureSelector } from '@ngrx/store';
+
+export const selectRouter = createFeatureSelector<RouterReducerState>('router');
```

Comment:
Using `getSelectors()` would also work if the default router configuration is used (`router` as key`). 

What if we add this line as a comment, and mention that this should only be required if the user deviates from the default config?

Reply:
Did it but not sure it's well written